pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -317,8 +317,14 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		if len(options.DNSIP) > 0 {
```

Comment:
No one has ever started node DNS by default, so only someone using the feature experimentally (outside of our control) would be in that setup.